src/App.vue

```diff
@@ -15,6 +15,105 @@
 import { defineComponent } from 'vue';
 
 export default defineComponent({
-    name: 'App'
+    name: 'App',
+
+    computed: {
+        updateAccountSession () {
+            return this.$store.state.account;
```

Comment:
`$store` is provided by which component? Should its origin be documented in a comment somewhere?

Reply:
Store is mounted to Vue at runtime so it's available everywhere that Vue is.